setup.py

```diff
@@ -229,8 +242,10 @@ def create_extension(ExtType, name, sources, dependencies, extra_link_args, extr
 class ONNXCpp2PyExtension(setuptools.Extension):
     def pre_run(self):
         self.sources = recursive_glob(SRC_DIR, '*.cc')
-        if os.path.join(SRC_DIR, "onnx-ml.pb.cc") in self.sources:
+        if (os.path.join(SRC_DIR, "onnx-ml.pb.cc") in self.sources and 0 == ONNX_ML):
```

Comment:
These should also check for onnx-operators.pb.cc and its ml variant; otherwise the linker will catch it and complain about duplicate symbols. More generally, it would also be nice to side-by-side some of this (so be able to build both projects in a single install), and to doc somewhere that ```pip install -e <dir> --install-option="--onnxml=1"``` is how the magic happens, but this change seems like a step in the right direction in terms of getting a good repeatable thing going.

Reply:
Yes agree, please pick out the correct 'pb.cc' file instead of raising errors.
And like above, do not use `0 == ONNX_ML`, it's a boolean already.

Reply:
sure, will add the build into regular build later and will also update the doc accordingly. Thanks!